os/drivers/video/video.c

```diff
@@ -1237,6 +1258,28 @@ static int video_poll(FAR struct file *filep, FAR struct pollfd *fds, bool setup
 	return OK;
 }
 
+static int video_set_stream_param(FAR video_upperhalf_t *priv, vbuf_container_t *dma_container, enum v4l2_buf_type buffer_type)
```

Comment:
good point.
But  if set_buftype & set_buf should be called together, you can remove them from video_devops_s
Let me hear your opinion.